interface/ceed-operator.c

```diff
@@ -1050,9 +1050,14 @@ int CeedOperatorSetQFunctionAssemblyDataUpdateNeeded(CeedOperator op, bool needs
   @ref Advanced
 **/
 int CeedOperatorSetNumQuadraturePoints(CeedOperator op, CeedInt num_qpts) {
-  CeedCheck(!op->is_composite, op->ceed, CEED_ERROR_MINOR, "Not defined for composite operator");
-  CeedCheck(op->num_qpts == 0, op->ceed, CEED_ERROR_MINOR, "Number of quadrature points already defined");
+  CeedCheck(!op->is_composite, op->ceed, CEED_ERROR_MINOR, "Not defined for composite CeedOperator");
   CeedCheck(!op->is_immutable, op->ceed, CEED_ERROR_MAJOR, "Operator cannot be changed after set as immutable");
+  if (op->num_qpts > 0) {
+    CeedCheck(num_qpts == op->num_qpts, op->ceed, CEED_ERROR_DIMENSION, "Different number of quadrature points already defined for the CeedOperator");
+    CeedWarn(
+        "CeedOperatorSetNumQuadraturePoints will be removed from the libCEED interface in the next release.\n"
+        "This function is reduntant and you can safely remove any calls to this function without replacing them.");
+  }
```

Comment:
Setting the wrong number of quadrature points is a bigger problem and I think a different one? I'm not sure. Ultimately probably fine either way since we will remove this in a few weeks

Reply:
> Setting the wrong number of quadrature points is a bigger problem and I think a different one?

Yeah, it's for sure a different problem, but one that doesn't *need* to be solved since we shouldn't be setting number of quadrature points to begin with.

Reply:
It might need to be solved though - if this number of quadrature points in this function call is used in other logic setting up an operator field

Reply:
My laptop is put away so I can't make the change (use the review to 'suggest a change' easily until Thursday)

Reply:
Whoops, missed the update message.